cmd/influxdb-writer/main.go

```diff
@@ -73,9 +72,16 @@ func main() {
 	}
 	defer client.Close()
 
-	repo := newService(client, repocfg, logger)
+	repo := influxdb.NewAsync(client, repocfg)
+
+	go func(log mflog.Logger) {
+		errCh := repo.Errors()
+		for err := range errCh {
+			log.Error(err.Error())
```

Comment:
Use `log.Error(err)` in case the writer publishes `nil` error to, say, indicate it is closed.

Reply:
log.Error() expects a string, so I put a err !=nil check